js/apps/account-ui/src/root/Root.tsx

```diff
@@ -35,9 +35,15 @@ const ReferrerLink = () => {
   ) : null;
 };
 
+const logoUrl = environment.logoUrl ? environment.logoUrl : "/";
+const internalLogoHref = useHref(logoUrl);
```

Comment:
I am not quite sure how this is works, as this also breaks the [Rules of Hooks](https://react.dev/warnings/invalid-hook-call-warning#breaking-rules-of-hooks), specifically:

> **Don’t call Hooks inside loops, conditions, or nested functions.** Instead, always use Hooks at the top level of your React function, before any early returns. You can only call Hooks while React is rendering a function component:
> 
> - ✅ Call them at the top level in the body of a function component.
> - ✅ Call them at the top level in the body of a custom Hook.

Since this call has been placed outside of a function component it violates these rules.